Library/Homebrew/unpack_strategy/tar.rb

```diff
@@ -38,8 +38,9 @@ def extract_to_dir(unpack_dir, basename:, verbose:)
         end
 
         system_command! "tar",
-                        args:    ["xf", tar_path, "-C", unpack_dir],
-                        verbose: verbose
+                        args:         ["xf", tar_path, "-C", unpack_dir],
+                        extra_params: extra_params||="",
```

Comment:
Why is the `||=""` needed?

Reply:
may I leave "||=" as it dose
but set it to empty array/ I need it because of super - inheritance in gnu_tar.rb

Reply:
I don't understand what you're saying here, please elaborate. Regardless this syntax is not correct and flagged by `brew style`

Reply:
Removed, now I'm using default value *extra_args: []* in the method arguments 
Ok, more accurate now with *brew style*